src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -115,21 +115,39 @@
    * Creates a new FirebaseUserManager instance.
    *
    * @param app A non-null {@link FirebaseApp}.
+   * @param tenantId The associated tenant ID if the user operations should be tenant-aware,
+   *     otherwise {@code null}
    */
-  FirebaseUserManager(@NonNull FirebaseApp app) {
+  FirebaseUserManager(@NonNull FirebaseApp app, @Nullable String tenantId) {
     checkNotNull(app, "FirebaseApp must not be null");
     String projectId = ImplFirebaseTrampolines.getProjectId(app);
     checkArgument(!Strings.isNullOrEmpty(projectId),
         "Project ID is required to access the auth service. Use a service account credential or "
             + "set the project ID explicitly via FirebaseOptions. Alternatively you can also "
             + "set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.");
-    this.userMgtBaseUrl = String.format(ID_TOOLKIT_URL, "v1", projectId);
+    if (tenantId == null) {
```

Comment:
`Strings.isNullOrEmpty` for good measure

Reply:
Hmm I don't think we want to check `Strings.isNullOrEmpty` in this case, since a null value for the tenant ID is acceptable.

However, if the value is non-null, I've added a check to ensure that it is not an empty string.

Reply:
What I meant was `if (String.isNullOrEmpty(tenantId))`. But this is ok too.

Reply:
Ah okay, that makes more sense!